test/OpenTelemetry.Instrumentation.EntityFrameworkCore.Tests/EntityFrameworkInstrumentationTests.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics;
+using Xunit;
+
+namespace OpenTelemetry.Instrumentation.EntityFrameworkCore.Tests;
+
+public class EntityFrameworkInstrumentationTests
```

Comment:
This test class is testing only itself. There is no reference to the _production_ code.

Reply:
removed